src/rand.jl

```diff
@@ -8,6 +9,17 @@ Base.rand(X::IntervalBox) = rand.(X)
 Random.gentype(::Type{Interval{T}}) where {T} = T
 Random.rand(rng::AbstractRNG, X::Random.SamplerTrivial{Interval{T}}) where {T} = rand(X[])
 
+Base.rand(type::Type{<:Interval}) = rand(GLOBAL_RNG, type)
+Base.rand(::Type{Interval}) = rand(Interval{Float64})
+
+function Random.rand(rng::AbstractRNG, ::Type{Interval{T}}) where {T}
```

Comment:
should this be `Random.randn` if it uses randn internally?

Reply:
I guess so, yes.

Reply:
I changed the name.